doc/man3/SSL_CTX_set_mode.pod

```diff
@@ -105,6 +105,16 @@ Enable asynchronous processing. TLS I/O operations may indicate a retry with
 SSL_ERROR_WANT_ASYNC with this mode set if an asynchronous capable engine is
 used to perform cryptographic operations. See L<SSL_get_error(3)>.
 
+=item SSL_MODE_NO_KTLS_TX
+
+Disable the use of the kernel TLS egress data-path. By default kernel TLS is
+enabled for supported ciphers. The kernel TLS data-path implements the record
```

Comment:
That's right. I'll add the bit about version to the documentation. Encrypt-than-mac is unrelated though. At the moment it is just AES-GCM TLS1.2.

Reply:
Does that mean encrypt-then-mac *is* supported by ktls?

Reply:
If we ever implement Encrypt-Than-MAC (which I highly doubt) then I think it would be supported with this API.

Reply:
Ah ok. But right now EtM is not supported, so KTLS must be disabled if it has been negotiated. We should ensure that something is documented about this. It could be kept fairly general, e.g. "By default kernel TLS is enabled if it is supported by the negotiated ciphersuites and extensions". Incidentally if EtM is not supported then this significantly reduces the usefulness of KTLS. My expectation (not based on any real data) is that the majority of TLSv1.2 connections will use it.

Reply:
Sure I'll fix the description here according to your suggestion.